spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe '#name_or_email' do
+    subject { user.name_or_email }
+
+    context '名前があるとき' do
+      let(:user) { create(:user) }
+
+      it { is_expected.to eq 'テスト 太郎' }
+    end
+
+    context '名前がないとき' do
+      let(:user) { create(:user, name: '') }
+
+      it { is_expected.to eq user.email }
```

Comment:
```suggestion
      let(:user) { create(:user, email: 'alice@example.com', name: '') }

      it { is_expected.to eq 'alice@example.com' }
```

ベタ書き推奨です。

https://speakerdeck.com/jnchito/number-vstat?slide=37

Reply:
`email` も追加しました。これで、[こちらのモヤモヤ](https://github.com/yuuu/fjord-books_app-2023/pull/1#discussion_r1340687898) も解消しました 👍